pkg/nsx/services/inventory/compare.go

```diff
@@ -88,3 +90,21 @@ func isIPChanged(pre containerinventory.ContainerApplicationInstance, cur contai
 	}
 	return false
 }
+
+func compareContainerApplication(pre interface{}, cur interface{}, property *map[string]interface{}) {
+	updateProperties := *property
+	if pre == nil {
+		updateProperties["display_name"] = cur.(containerinventory.ContainerApplication).DisplayName
```

Comment:
It's not specific to this PR, but I'm thinking about maybe we can define those at the beginning to reduce some duplication of type converting? This may also apply to other compare function.
```
curApplication := cur.(containerinventory.ContainerApplication)
preApplication := pre.(containerinventory.ContainerApplication)
```

Reply:
@TaoZou1 +1